onnx_tf/handlers/backend/cumsum.py

```diff
@@ -0,0 +1,45 @@
+import tensorflow as tf
+
+from onnx_tf.common import exception
+from onnx_tf.handlers.backend_handler import BackendHandler
+from onnx_tf.handlers.handler import onnx_op
+from onnx_tf.handlers.handler import partial_support
+from onnx_tf.handlers.handler import ps_description
+from onnx_tf.handlers.handler import tf_func
+
+
+@onnx_op("CumSum")
+@tf_func(tf.math.cumsum)
+@partial_support(True)
+@ps_description(
+    "CumSum inputs in uint32/uint64 " + "are not supported in Tensorflow."
+)
+class CumSum(BackendHandler):
+  @classmethod
+  def args_check(cls, node, **kwargs):
+    supported_dtype = [
+        tf.bfloat16, tf.half, tf.float32, tf.float64, tf.uint8, tf.uint16,
+        tf.int8, tf.int16, tf.int32, tf.int64, tf.complex64, tf.complex128
+    ]
+    x = kwargs["tensor_dict"][node.inputs[0]]
+    if x.dtype not in supported_dtype:
+      exception.OP_UNSUPPORTED_EXCEPT(
+          "CumSum input in " + str(x.dtype) + " which", "Tensorflow")
+
+  @classmethod
+  def version_11(cls, node, **kwargs):
+    tensor_dict = kwargs["tensor_dict"]
+    x = tensor_dict[node.inputs[0]]
+    inputs = [x]
+
+    if len(node.inputs) > 1:
+      # optional 0-D tensor, range [-rank(x), rank(x)-1]
+      axis = tensor_dict["axis"]
+      inputs.append(tf.reduce_min(axis))
+
```

Comment:
tf.reduce_min doesn't seem necessary here. The input axis is already a 0-D tensor, meaning the number of elements always 1.

Reply:
@pluradj can you check whether tf.reduce_min is needed and provide comments? Thanks!

Reply:
You're right @chinhuang007 -- the input is declared in the spec to be a 0-D tensor. I removed the call to `tf.reduce_min`.

The remaining problem is that the test cases in the `ONNX` repository were passing a 1-D tensor. I've pushed a fix to https://github.com/onnx/onnx/pull/2611 to fix that issue. Unfortunately, the CumSum `test_onnx_backend.py` tests won't pass until that PR is merged. I've tested with both PRs locally, and the backend test will pass.